README.md

```diff
@@ -329,6 +329,28 @@ class Product {
 }
 ```
 
+If you have a list of acceptable status codes you can also pass an array to the
+```statusCode``` option:
+
+```php
+<?php
+namespace CircleBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Circle\DoctrineRestDriver\Annotations as DataSource;
+
+/**
+ * This annotation marks the class as managed entity:
+ *
+ * @ORM\Entity
+ * @ORM\Table("products")
+ * @DataSource\Select("http://www.yourSite.com/api/products/findOne/{id}", statusCode={200, 203, 404})
```

Comment:
Hey sorry for responding so late. I just needed a bit time to think about it. Well I don't WANT it to be a new major version, but I don't want an API that is stange just to ensure backwards compatibility. So let's rename it and upgrade the major version. We should mention it in the release.